chapters/image_accceses.md

```diff
@@ -0,0 +1,146 @@
+# Image Accesses
+
+SPIR-V has various [Image Instructions](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_image_instructions) that interact with `OpTypeImage`. This chapter aims to give some extra context around these instructions.
+
+> There is an [example.frag](./examples/image_accesses/example.frag) (and [example.spv](./examples/image_accesses/example.spv)) GLSL shader that has an example of various ways images can be accessed in SPIR-V.
+
+# Coordinate Systems
+
+It is important to know there are 3 texel coordinate systems used for images:
+
+- Normalized (floats)
+- Unnormalized (floats)
+- Integers
+
+The SPIR-V spec documents which instructions are allowed to use which coordinate system.
+The [Vulkan Spec Texel Coordinate Systems Chapter](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#textures-texel-coordinate-systems) is a good resource to learn more about this.
+
+When using a `vec4` (4 wide vector) as a coordinate, the API spec defines which each vector element points to. As an example, in Vulkan, when using Unnormalized coordinates, the `vec4.z` element is the value of the `array layer`.
+
+# Images as Handles
+
+An `OpTypeImage` can be thought of as a handle to the underlying texel data. To help illustrate this, here is a small program that writes to an image
+
+```glsl
+layout(set = 0, binding = 1, rgba8) uniform writeonly image2D image;
+void main() {
+    imageStore(image, ivec2(0, 0), vec4(1.0));
+}
+```
+
+Taking a closer look at the `OpVariable` we see it is of type [UniformConstant](../chapters/storage_class.md#uniformconstant) which is suppose to be "read-only" (yet there is a clear `OpImageWrite` occuring here).
```

Comment:
Nitpick: it's better to say that the variable is in the UniformConstant storage class, or that it points to the UniformConstant storage class.  The actual type of the OpVariable is an OpTypePointer.

https://godbolt.org/z/Md6WE7qEx

Reply:
> https://godbolt.org/z/Md6WE7qEx

:100: